### PR TITLE
use oneCycleLR scheduler to fine tune model optimizer

### PR DESCRIPTION
**Pull request by kaixuanliu**
With the help of this optimizer scheduler and large learning rate, we can get higher accuracy when using large batch size to do the distributed training. Experiment shows we can get 72.15 for 16 partitions igbh-full training with local bs=4096, which can help us drop the epoch time from 18 hours to 12 hours. Attached is the experiment results:

image

### CODE REVIEW

**Comment by kaixuanliu** [comment]
Discard it temporarily.
